src/content/docs/security/security-privacy/compliance/data-encryption.mdx

```diff
@@ -68,15 +68,15 @@ All New Relic customers benefit from the security provided with data encryption
   </tbody>
 </table>
 
-## Disk encryption
+## Encryption at Rest
 
-New Relic's disk-based encryption provides additional security while your data is at rest (FIPS 140-2 compliant). For more information, see the documentation about our [Telemetry Data Platform](/docs/telemetry-data-platform/ingest-manage-data/get-started/get-know-telemetry-data-platform) and [managing data retention](/docs/telemetry-data-platform/ingest-manage-data/manage-data/manage-data-retention).
+New Relic ensures the security of your data at rest with (FIPS 140-2 compliant) cryptography. For more information, see more documentation about our [Telemetry Data Platform](/docs/telemetry-data-platform/ingest-manage-data/get-started/get-know-telemetry-data-platform) and [managing data retention](/docs/telemetry-data-platform/ingest-manage-data/manage-data/manage-data-retention).
```

Comment:
Can we say in the first sentence:
When New Relic encrypts dat at rest, New Relic does so in accordance with FIPS 140-2 standards.

Reply:
Done.